src/gdal_utils.cpp

```diff
@@ -247,6 +247,44 @@ Rcpp::LogicalVector CPL_gdaltranslate(Rcpp::CharacterVector src, Rcpp::Character
 	return result == NULL || err;
 }
 
+
+// [[Rcpp::export]]
+Rcpp::LogicalVector CPL_gdalfootprint(Rcpp::CharacterVector src, Rcpp::CharacterVector dst,
+        Rcpp::CharacterVector options, Rcpp::CharacterVector oo, Rcpp::CharacterVector co,
+        bool quiet = true) {
+	
+#if GDAL_VERSION_NUM < 3080000
+	Rcpp::CharacterVector ret;
+	Rcpp::stop("gdalfootprint util requires GDAL >= 3.8.0");
+#endif
```

Comment:
This needs to be an `#else`, and the `#endif` should go after line 283.

Reply:
Thanks! I think I now use `#if` properly.